orttraining/orttraining/models/bert/main.cc

```diff
@@ -161,6 +161,8 @@ Status ParseArguments(int argc, char* argv[], BertParameters& params, OrtParamet
       "seperated by ':'. If consumer nodes need to be specified, specify them after producer node with a '-' delimiter and "
       "separate each consumer node with a '/'. ", cxxopts::value<std::vector<std::string>>()->default_value(""))
       ("enable_grad_norm_clip", "Specify whether to enable gradient clipping for optimizers.",
+        cxxopts::value<bool>()->default_value("true"))
+      ("enable_gelu_approximation", "Specify whether to enable GELU approximation.",
```

Comment:
is this for bert only?

Reply:
at the moment, yes :)

---
In reply to: [445263626](https://github.com/microsoft/onnxruntime/pull/4146#discussion_r445263626) [](ancestors = 445263626)